consoles/vnc_base.pm

```diff
@@ -188,9 +188,11 @@ sub mouse_hide {
     $self->{mouse}->{x} = $self->{vnc}->width - 1;
     $self->{mouse}->{y} = $self->{vnc}->height - 1;
 
-    my $border_offset = int($args->{border_offset});
```

Comment:
you could have fixed this by just passing a || 0 within the int cast - the -= 0 wouldn't have hurt. 

But of course your version is spelling it out with more whitespace :)